geomesa-utils/src/main/resources/org/locationtech/geomesa/utils/io/remote-prefixes.list

```diff
@@ -0,0 +1,6 @@
+adl
+hdfs
+s3n
+s3a
```

Comment:
I almost want to just convert s3 to s3a via configuration :)

Reply:
also, should these be loaded via site.xml files?

Reply:
hmm, i just copied these from the list we had in code - i figured this way we could at least add new ones without recompiling.